frontend/packages/shared/src/hooks/useUpdateBpmn.test.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { useUpdateBpmn } from './useUpdateBpmn';
+import { queriesMock } from '../mocks/queriesMock';
+import { app, org } from '@studio/testing/testids';
+import { getDataTypesToSignMock } from '../mocks/bpmnDefinitionsMock';
+import { removeDataTypeIdsToSign, updateDataTypeIdsToSign } from '../utils/bpmnUtils';
+import type { ServicesContextProps } from '../contexts/ServicesContext';
+import { ServicesContextProvider } from '../contexts/ServicesContext';
+import { renderHook } from '@testing-library/react';
+
+const dataTypesToSignMock = getDataTypesToSignMock(['dataType1', 'dataType2']);
+const moddle = {
+  fromXML: jest.fn().mockResolvedValue({
+    rootElement: dataTypesToSignMock,
+  }),
+  toXML: jest.fn().mockResolvedValue({ xml: '<newXml></newXml>' }),
+};
+jest.mock('bpmn-moddle', () => jest.fn(() => moddle));
+
+const mockBPMNXML: string = `<?xml version="1.0" encoding="UTF-8"?></xml>`;
+
+describe('useUpdateBpmn', () => {
+  afterEach(jest.clearAllMocks);
+
+  describe('removeDataTypeIdsToSign', () => {
+    it('update the bpmn file if the deleted data type ids are present', async () => {
+      const { result } = renderUpdateBpmnHook();
+
+      await result.current(removeDataTypeIdsToSign(['dataType1']));
+
+      expect(moddle.fromXML).toHaveBeenCalledWith(mockBPMNXML);
+      expect(moddle.toXML).toHaveBeenCalledWith(dataTypesToSignMock, { format: true });
+      expect(
+        dataTypesToSignMock.rootElements[0].flowElements[0].extensionElements.values[0].$children[0]
+          .$children[0].$children[0].$body,
```

Comment:
Can this be put inside a named variable?

Reply:
Done 👍